cypari2/convert.pyx

```diff
@@ -145,14 +150,17 @@ cpdef gen_to_integer(Gen x):
 
     Tests:
 
-    >>> for i in range(10000):
-    ...     x = 3**i
-    ...     if long(pari(x)) != long(x):
-    ...         print(x)
-    >>> gen_to_integer(pari("1.0 - 2^64"))
-    -18446744073709551615L
-    >>> gen_to_integer(pari("1 - 2^64"))
-    -18446744073709551615L
+    >>> gen_to_integer(pari("1.0 - 2^64")) == -18446744073709551615
+    True
+    >>> gen_to_integer(pari("1 - 2^64")) == -18446744073709551615
+    True
+    >>> import sys
+    >>> if sys.version_info.major == 2:
```

Comment:
The same test should be done with `int` instead of `long` (and you might as well do it on Python 2 and Python 3).